api/src/main/resources/workbench.yaml

```diff
@@ -4320,3 +4343,52 @@ definitions:
         type: "array"
         items:
           $ref: "#/definitions/Institution"
+
+  AdminFederatedWorkspaceDetailsResponse:
```

Comment:
Naming nit: I'm not in love with the "federated" interstitial word there. I think the clarity of naming wouldn't be meaningfully affected if you dropped it. – AdminWorkspaceDetailsResponse.